.github/workflows/test.yml

```diff
@@ -72,6 +75,12 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: test
+      - name: cargo test with musl
+        if: matrix.os == 'ubuntu-latest'
+        uses: actions-rs/cargo@v1
+        with:
+          command: test
+          args: --target x86_64-unknown-linux-musl
```

Comment:
Since we're only shipping musl anyway, wouldn't it make more sense to only run the tests for musl?

Reply:
Just in case someone needs to compile maturin from source instead of using prebuilt wheels, for example on some BSD systems?